src/Mix.php

```diff
@@ -27,6 +27,12 @@ class Mix
      */
     public function __invoke($path, $manifestDirectory = ''): string
     {
+        $urlDomain = '';
+        if (preg_match('@^([a-z]+://[^/]+)(/.+)$@', $path, $matches)) {
```

Comment:
Can you please add a comment saying what this regex is doing? Also, it would be great if you also add a https://regex101.com/ URL.

Reply:
Comment added.